test/integration/guest_user_test.rb

```diff
@@ -144,14 +144,16 @@ class GuestUserTest < ActionDispatch::IntegrationTest
     end
   end
 
-  test "an unauthorized user can add projects to pending_loans but cannot
-  checkout page logging in" do
-    tenant = create(:tenant)
-    tenant.projects << create(:project, price: 8900)
-
+  test "after an unauthorized user adds items and clicks checkout and logs in,
```

Comment:
I think this test doesn't quite match the name...I would suggest moving the body of this test to a user_test file, and adding another test that actually does test "after an unauthorized user adds items and clicks checkout and logs in, the pending_loans still retains the items"